addon/styles/addon.scss

```diff
@@ -52,6 +54,12 @@ $frost-modal-overflow-shadow-color: rgba(0, 0, 0, .13);
   min-height: 250px;
   max-height: calc(100vh - 90px); // TODO Nav height variable
 
+  &.frost-modal-form {
```

Comment:
I'm still back to the same thing - why this extra layer of class selector?  Should it either be wrapped around everything or not used at all?

Reply:
Because it's a parent class The only context given if it's a form or not is from the binding which can be passed to `to-elsewhere` modal.

![image](https://user-images.githubusercontent.com/9057680/37622766-0146a370-2b99-11e8-9b67-826b9cd24e94.png)

Reply:
There was already a `frost-modal-dialog` class being applied to the same element and the stylesheets were configured around that.  So there was no need to add the `frost-modal-form` class selector.  I'm fine that you did, but once you did then why are we not updating the other class definitions to leverage it as well.  Now we have different elements being referenced in different ways, when what has introduced this difference is the change you have made.

Reply:
Not sure I follow. The reason for the `frost-modal-form` selector is because UX wants the form and **only** that form modal to have that border on its header. The other modals stay the same